### ISSUE TITLE
[Feature Request]  Support pulling details directly from kubernetes specs

### ISSUE THREAD

**Issue by cornfeedhobo** [norwoodj/helm-docs#129]
I'm finding myself doing stuff like

```yaml
# -- NodeSelector is a selector which must be true for the pod to fit on a node.
# Selector which must match a node's labels for the pod to be scheduled on that node.
# More info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
nodeSelector: {}
```

This is a direct copy from the StatefulSet API docs, and I have a number of other fields that are the same.

I think it would be neat to have references that get resolved at generation time.

**Comment by norwoodj** [comment]
I'm not sure what you mean by this exactly. Are you proposing this information gets pulled from the page you've linked?

**Comment by cornfeedhobo** [comment]
@norwoodj yeah, that's essentially what I'm suggesting - a way to say "copy the text from the official spec", which I would hope is available somewhere as markdown that is structured and can be parsed.

**Comment by Nepo26** [comment]
I think a way to better handle this would be creating a way to add code in the process. Maybe a plugin interface or similar. We need to discuss this feature more to make it more straightforward.

**Comment by cornfeedhobo** [comment]
In case this ever gets done, I propose a simple interface:

```yaml
statefulset:
  # -- ``statefulset.spec.podManagementPolicy``
  podManagementPolicy: ""
```

Then the documentation generator could pull the description from somewhere like [the k8s documentation website](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/stateful-set-v1/#StatefulSetSpec), and link to it.
